addon/components/uni-button.js

```diff
@@ -4,7 +4,9 @@ import layout from '../templates/components/uni-button';
 const { Component } = Ember;
 
 export default Component.extend({
-  tagName: 'button',
+  tagName: 'div',
+  attributeBindings: ['aria-role'],
+  'aria-role': 'button',
```

Comment:
I don't know if it is the same, but Ember has a special property for the aria-role: http://emberjs.com/api/classes/Ember.AriaRoleSupport.html#property_ariaRole